demo/Script/FmodTest.gd

```diff
@@ -5,7 +5,7 @@ func _enter_tree():
 	# set up FMOD
 	Fmod.set_software_format(0, Fmod.GD_FMOD_SPEAKERMODE_STEREO, 0)
 	Fmod.init(1024, Fmod.GD_FMOD_STUDIO_INIT_LIVEUPDATE, Fmod.GD_FMOD_INIT_NORMAL)
-	Fmod.set_sound_3D_settings(1, 32, 1)
+	Fmod.set_sound_3D_settings(1.0, 32.0, 1.0)
```

Comment:
It looks like there is a bug when passing integers instead of float that the value is 0 for some reason. I changed it now to float but in future we might want to look into this.